src/main/java/seedu/duke/commands/SelectPatientCommand.java

```diff
@@ -34,8 +34,14 @@ public CommandResult execute() {
 
         try {
             String patientName = hospital.getPatient(index).getName();
-            String resultMessage = String.format(MESSAGE_SUCCESS, patientName);
+            String patientTag = hospital.getPatient(index).getTag();
+            String formattedTag = "";
+            if (patientTag != null){
+                formattedTag = " [" + patientTag + "]";
+            }
+            String resultMessage = String.format(MESSAGE_SUCCESS, patientName + formattedTag);
```

Comment:
Same, you can make use of the patient object. 

```
Patient p = hospital.getPatient(index)

String resultMessage = String.format(MESSAGE_SUCCESS, p.name + p.getFormattedTag);
```

Reply:
good idea
will change it to the suggested implementation